Makefile

```diff
@@ -43,6 +43,14 @@ test-integration: install
 	docker-compose -f dev/iceberg-integration/docker-compose-integration.yml exec -T spark-iceberg ipython ./provision.py
 	venv/bin/python -m pytest deltacat/tests/integ -v -m integration
 
+test-converter:
```

Comment:
This is fine for now (doesn't conflict with any other test names), but we'll probably want to change its name or roll it up within an existing suite (like integ tests) later.